src/styles/blocks/page.scss

```diff
@@ -0,0 +1,38 @@
+.page {
+  font-family: 'Space Grotesk', sans-serif;
+  font-weight: 700;
```

Comment:
It's better to set the font-weight to normal (400) by default. If needed, you can change the font-weight for individual elements later.